src/Tag.php

```diff
@@ -60,6 +60,22 @@ public static function getWithType(string $type): DbCollection
         return static::withType($type)->ordered()->get();
     }
 
+    public static function find(
```

Comment:
This is not only a tag class it is also an _Eloquent_ model, rewriting the _find_ method and making it behave differently to the parent class will IMO create very confusing gotchas for anyone using this package. The _find_ method generally accepts the PK of the table as a parameter, this rewrites the way _Eloquent_ works.

Reply:
Indeed! I totally forgot about that :raised_hands: Thanks for pointing that out